javascript/ql/src/semmle/javascript/GeneratedCode.qll

```diff
@@ -121,6 +121,27 @@ private predicate isGeneratedHtml(File f) {
     e.getName() = "meta" and
     e.getAttributeByName("name").getValue() = "generator"
   )
+  or
+  20 < countStartingHtmlElements(f, _)
+}
+
+/**
+ * Gets an element that starts at line `l` in file `f`.
+ */
+private HTML::Element getAStartingElement(File f, int l) {
+  result.getFile() = f and result.getLocation().getStartLine() = l
+}
+
+
+/**
+ * Gets the number of HTML elements that starts at line `l` in file `f`.
+ */
+private int countStartingHtmlElements(File f, int l) {
+  exists(HTML::Element candidate |
+    candidate.getFile() = f and
+    candidate.getLocation().getStartLine() = l and
+    result = count(getAStartingElement(f, l))
```

Comment:
...and isn't this whole predicate just `result = strictcount(getAStartingElement(f, l))`?